test/tools/d_do_test.d

```diff
@@ -610,6 +610,9 @@ int tryMain(string[] args)
     string output_file    = result_path ~ input_file ~ ".out";
     string test_app_dmd_base = output_dir ~ envData.sep ~ test_name ~ "_";
 
+    if (test_base_name.endsWith("_dshell.d"))
```

Comment:
I recommend putting these the .d files in a different directory instead.

Reply:
done